README.md

```diff
@@ -190,6 +194,110 @@ pip install ni-measurement-plugin-sdk-generator
 
 ---
 
+## Generating Measurement Clients: Quick Start
+
+This section provides instructions to generate custom measurement clients in Python using Measurement Plug-In SDK for Python.
+
+### Installation
+Install Measurement Plug-In SDK by following the instructions in section ["Measurement Plug-In SDK Installation"](#installation).
+
+### Generating a minimal Python measurement client
+
+1. Install the `ni-measurement-plugin-sdk-generator` package.
+
+    ``` cmd
+    REM Activate the required virtual environment if any.
+    pip install ni-measurement-plugin-sdk-generator
+    ```
+
+2. Run the command `ni-measurement-plugin-client-generator` tool. Use either of the below modes to create Python measurement clients.
+
+    1. Run this command with optional arguments to create measurement clients for specific measurements.
+
+        `ni-measurement-plugin-client-generator --measurement-service-class "ni.examples.SampleMeasurement_Python" --module-name "sample_measurement"  --class-name "SampleMeasurement" 
+        --directory-out <new_path_for_created_files>`
+
+        - `--measurement-service-class` specifies the measurement service class for which the client is being generated.
+
+        - `--module-name` and `--class-name` define the module and class names of the generated client. If not specified, they are derived from the measurement service class name.
+
+        - `--directory-out` specifies the output directory for the generated files. If not provided, files are placed in the current directory.
+        
+        > **Note**: For multiple measurement client creation, `--module-name` and `--class-name` are ignored, and they are derived from service class for each measurement. So, ensure measurement service class name adheres to proper naming conventions.
+
+    2. Run this command to create measurement clients for all registered measurements.
+
+        `ni-measurement-plugin-sdk-generator --all`
+
+        > **Note**: `--directory-out` can be provided for this mode.
+
+    3. Run this command to interactively create measurement clients for any registered measurements.
+
+        `ni-measurement-plugin-sdk-generator --interactive`
+
+3. The genarted client comprises four APIs: `measure`, `stream_measure`, `register_pin_map`, and `cancel`. The usage of these APIs is discussed in the section ["Steps to run/debug the measurement client".](#steps-to-rundebug-the-measurement-client)
+
+> **Note**:
+> - The Measurement Plug-In Client is compatible with all datatypes supported by the Measurement Plug-In.
+> - Measurement configuration of Double XY datatype is not supported. 
```

Comment:
The Double XY datatype is not supported for measurement configurations (inputs).